dags/glam.py

```diff
@@ -0,0 +1,148 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from utils.gcp import bigquery_etl_query
+from airflow.operators.sensors import ExternalTaskSensor
+
+
+dataset_id = "telemetry_derived"
+default_args = {
+    'owner': 'msamuel@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2019, 10, 22),
+    'email': ['telemetry-alerts@mozilla.com', 'msamuel@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=30),
+}
+
+# Make sure all the data for the given day has arrived before running.
+# Running at 1am should suffice.
+dag = DAG('glam', default_args=default_args, schedule_interval='@daily')
+
+wait_for_main_ping = ExternalTaskSensor(
+    task_id="wait_for_main_ping",
+    external_dag_id="main_summary",
+    external_task_id="copy_deduplicate_main_ping",
+    execution_delta=timedelta(hours=-1),
+    check_existence=True,
+    dag=dag,
+)
+
+clients_daily_scalar_aggregates = bigquery_etl_query(
+    task_id="clients_daily_scalar_aggregates",
+    destination_table="clients_daily_scalar_aggregates_v1",
+    dataset_id=dataset_id,
```

Comment:
```suggestion
    dataset_id=dataset_id,
    project_id='moz-fx-data-shared-prod',
```

All of the query tasks will need to have project set to `moz-fx-data-shared-prod` so that they write into that project rather than the default derived-datasets project.